src/display/MovablePointDisplay.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react"
+import { vec } from "../vec"
+import { useTransformContext } from "../context/TransformContext"
+import { Theme } from "./Theme"
+
+export interface MovablePointDisplayProps {
+  color?: string
+  ringRadiusPx?: number
```

Comment:
I like the `px` suffix to differentiate it from Mafs-space. I would like to standardize this.